extras/usd/tutorials/endToEnd/tutorial_scripts/add_shadingVariants.py

```diff
@@ -93,6 +93,7 @@ def _AddShadingToBall(stage):
         with shadingVariant.GetVariantEditContext():
             whichBall = variantName.split('_')[-1]
             texPath = os.path.join(texDir, 'ball%s.tex' % whichBall)
+            texPath = texPath.replace('\\', '/')
             # in the current variant, modify the color
```

Comment:
Sure thing! Anyway, I'm not suggesting that a merge be gated by my comment. It's only a comment ;)